graylog2-web-interface/src/components/indexers/IndexerClusterHealth.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React, { useEffect, useState } from 'react';
+
+import { Spinner } from 'components/common';
+import { Row, Col } from 'components/bootstrap';
+import { DocumentationLink, SmallSupportLink } from 'components/support';
+import DocsHelper from 'util/DocsHelper';
+import { IndexerClusterHealthSummary } from 'components/indexers';
+import { IndexerClusterActions } from 'stores/indexers/IndexerClusterStore';
+
+import IndexerClusterHealthError from './IndexerClusterHealthError';
+
+const useLoadHealthAndName = () => {
+  const [state, setState] = useState({ health: undefined, name: undefined, error: undefined, loading: false });
+
+  useEffect(() => {
+    setState({ health: undefined, name: undefined, error: undefined, loading: true });
+
+    Promise.all([
+      IndexerClusterActions.health(),
+      IndexerClusterActions.name(),
```

Comment:
Since we fetch the name, let's display it in the UI (when it exists), although this has not been the case before.
It looks like this is only possible, when the cluster is healthy. We could include it in `Elasticsearch cluster "the-name" is green ..`.